compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -2659,7 +2663,10 @@ pub(crate) enum DiagnosticMode {
     /// The binding is part of a pattern
     Pattern,
     /// The binding is part of a use statement
-    Import,
+    Import {
+        /// Is the format `use a::{b,c}`
+        single_nested: bool,
```

Comment:
```suggestion
        append: bool,
```
To emphasize how we change the diagnostic (from replacing to appending), instead of the initial reason for that.